src/amber/websockets/adapters/redis.cr

```diff
@@ -10,8 +10,8 @@ module Amber::WebSockets::Adapters
 
     # Establish subscribe and publish connections to Redis
     def initialize
-      @subscriber = Redis.new(url: Amber::Server.redis_url)
-      @publisher = Redis.new(url: Amber::Server.redis_url)
+      @subscriber = Redis.new(url: Amber::Server.settings.redis_url)
```

Comment:
Weren't you planning on moving all of these straight to Amber like `Amber.settings.redis_url`.

Reply:
Just did 😃